Source/Atomic/UI/UI.h

```diff
@@ -100,6 +100,8 @@ class UI : public Object, private tb::TBWidgetListener
     void ShowDebugHud(bool value);
     void ToggleDebugHud();
 
+    void CycleDebugHudMode();
```

Comment:
When adding new methods, I have been trying to remember to add a simple one or two line comment

```/// Cycle between showing rendering stats on the debug hud (none, rendering stats, stats and mode, stats and profiler) 

These are used for generated API reference documentation.  We don't have a coding standard document drafted yet.  Though, I want to get to this really soon.  I think requiring comments on public API is a good one.